Mathlib/Topology/Algebra/ContinuousAffineMap.lean

```diff
@@ -75,25 +75,22 @@ theorem congr_fun {f g : P →ᴬ[R] Q} (h : f = g) (x : P) : f x = g x :=
 def toContinuousMap (f : P →ᴬ[R] Q) : C(P, Q) :=
   ⟨f, f.cont⟩
 
--- Porting note: changed to CoeHead due to difficulty with synthesization order
 instance : CoeHead (P →ᴬ[R] Q) C(P, Q) :=
   ⟨toContinuousMap⟩
 
 @[simp]
 theorem toContinuousMap_coe (f : P →ᴬ[R] Q) : f.toContinuousMap = ↑f := rfl
 
-@[simp] -- Porting note: removed `norm_cast`
```

Comment:
These porting notes are picking up on an actual problem. The person porting this didn't mark `ContinuousAffineMap.toAffineMap` with the `@[coe]` attribute (probably in part because this function is declared via the `extends` keyword). So when `norm_cast` looks at this lemma it sees a single coercion on each side of `=` (instead of two on the LHS and one on the RHS). I'll fix this in a another PR, but please undo the deletion of these three notes (the one above synth order can still be removed).

Reply:
Done in #22410

Reply:
Reverted.